healthd/healthd_mode_charger.cpp

```diff
@@ -93,9 +93,9 @@ char* locale;
 
 #define LAST_KMSG_MAX_SZ (32 * 1024)
 
-#define LOGE(x...) KLOG_ERROR("charger", x);
-#define LOGW(x...) KLOG_WARNING("charger", x);
-#define LOGV(x...) KLOG_DEBUG("charger", x);
+#define LOGE(x...) KLOG_ERROR("charger", x); fprintf(stderr,x);
```

Comment:
That's not really related to the commit, but fine

Reply:
Yes, and it is the only way to debug